FreeAPS/Sources/Services/Network/NightscoutAPI.swift

```diff
@@ -5,7 +5,7 @@ import Foundation
 class NightscoutAPI {
     init(url: URL, secret: String? = nil) {
         self.url = url
-        self.secret = secret
+        self.secret = (secret ?? "").isEmpty ? nil : secret!
```

Comment:
`self.secret = secret?.nonEmpty`

Reply:
Haven't seen that helper class. Looks much cleaner now :)